### ISSUE TITLE
Results not reproducable? 

### ISSUE THREAD

**Issue by manoja328** [jinyu121/CIOD#1]
Hello CIOD authors, 

I was trying to  run your code on VOC but couldnot reporduce the results with the command that was given in the repo. Are there any specific hyperparameters that I need to change to reporduce the results in the  paper. With the deafult settings I only get the following for VOC: 

========== RCNN Test set Summary (mAP, AP, %) NOREPR  NEW ==========
66.77 :	76.27	77.38	71.37	61.54	47.27	
58.15 :	48.92	69.31	53.01	38.05	25.31	67.56	82.25	80.12	43.44	73.47	
32.62 :	12.42	17.92	20.97	15.87	16.40	42.24	53.29	20.44	15.03	22.77	36.49	49.47	69.63	24.23	72.08	
25.76 :	12.61	11.40	10.18	10.13	6.16	7.87	39.36	31.07	6.06	5.07	5.70	21.64	53.77	35.47	52.51	29.91	31.24	52.66	42.50	49.89	
========== RCNN Test set Group Summary (mAP, AP, %) NEW NOREPR ==========
Group  0 : 66.77	->	46.92	->	16.72	->	10.09
Group  1 : 69.37	->	30.75	->	17.89
Group  2 : 50.38	->	33.82
Group  3 : 41.24

**Comment by jinyu121** [comment]
Hi~

Please try to checkout commit `d58255605e37e362ee746637fe93efb669a7e7ab`.

Actually, no tricks, no specific hyper-parameters.

**Comment by wassryan** [comment]
Hi, I checkout commit `d58255605e37e362ee746637fe93efb669a7e7ab`, and the result cannot reproduce either.
========== RCNN Test set Summary (mAP, AP, %) NOREPR  NEW ==========
55.99 : 68.03   77.50   65.89   42.34   26.19
51.84 : 32.66   46.17   44.88   32.98   20.19   66.97   77.00   84.69   38.08   74.83
42.74 : 20.14   30.03   34.22   24.78   25.12   55.58   39.10   62.23   18.98   28.70   31.66   70.8068.01    60.91   70.80
40.17 : 18.12   28.53   36.02   25.88   22.35   17.68   49.11   50.75   20.00   29.34   18.67   57.6067.06    54.68   60.44   28.74   45.39   57.46   56.05   59.46
========== RCNN Test set Group Summary (mAP, AP, %) NEW NOREPR ==========
Group  0 : 55.99        ->      35.38   ->      26.86   ->      26.18
Group  1 : 68.31        ->      40.92   ->      33.38
Group  2 : 60.44        ->      51.69
Group  3 : 49.42

which has a huge gap with the result in paper. Can you reproduce the result on your server and help me figure out the problem? Thanks!
